### PR TITLE
[FIX] Cleanup code

### PR DESCRIPTION
**Pull request by im-kulikov**
- part of code is outdated
- part of tests is outdated

I can't test code, because it depends on files/services that not in project..
And, some of tests are out of date...

### CODE REVIEW

**Comment by im-kulikov** [comment]
@c-darwin any updates?

File: packages/api/smart.go [hunk 1]
```diff
@@ -78,8 +78,8 @@ func validateSmartContract(r *http.Request, data *apiData, result *prepareResult
 					pref := getPrefix(data)
 					signature := &model.Signature{}
 					signature.SetTablePrefix(pref)
-					found, err := signature.Get(ret[1])
-					if err != nil {
+					var found bool
```
Comment (ghost) [inline packages/api/smart.go:81]: what is the reason for explicit variable declaration?

File: packages/api/vde.go [hunk 2]
```diff
@@ -102,32 +102,32 @@ func InitSmartContract(sc *smart.SmartContract, data []byte) error {
...
 				var b []byte
-				if err := converter.BinUnmarshal(&input, &b); err != nil {
+				if err = converter.BinUnmarshal(&input, &b); err != nil {
 					return err
 				}
 				v = hex.EncodeToString(b)
 			case `[]interface {}`:
-				count, err := converter.DecodeLength(&input)
-				if err != nil {
+				var count int64
```
Comment (ghost) [inline packages/api/vde.go:122]: One more explicit variable declaration.

File: packages/api/vde.go [hunk 3]
```diff
@@ -102,32 +102,32 @@ func InitSmartContract(sc *smart.SmartContract, data []byte) error {
...
+				if count, err = converter.DecodeLength(&input); err != nil {
 					return err
 				}
 				isforv = true
 				list := make([]interface{}, 0)
 				for count > 0 {
-					length, err := converter.DecodeLength(&input)
-					if err != nil {
+					var length int64
```
Comment (ghost) [inline packages/api/vde.go:129]: the same thing

**Review by ghost — COMMENTED** [review]
So it is a lot of unnecessary explicit variable declarations. It isn't quite clear the goal of changing
`result := []*QueueTx{}` to `var result []*QueueTx` and others declarations. 

Futhermore, don't mix several goals in one commit please. If you want to fix abbreviations naming such as Id -> ID do it in a separate commit from the commint where you fix error message.

File: packages/api/smart.go (hunk 1 above)
Comment (im-kulikov) [reply packages/api/smart.go:81]: `err` shadowing

File: packages/api/vde.go (hunk 2 above)
Comment (im-kulikov) [reply packages/api/vde.go:122]: `err` shadowing

File: packages/api/vde.go (hunk 3 above)
Comment (im-kulikov) [reply packages/api/vde.go:129]: `err` shadowing

**Pull request closed without merging**